tests/test_flows.py

```diff
@@ -851,6 +851,28 @@ def test_topology(self):
         debug("Confirming topology query was received")
         self.check_log(env.agents[0], r"TOPOLOGY_QUERY_MESSAGE.*mid={:d}".format(mid))
 
+    def test_beacon_report_query(self):
+        # associated STA
+        sta = env.Station.create()
+
+        # for testing non existing STA, when want to test the error flow
+        # sta1 = env.Station.create()
+
+        debug("Connect dummy STA (" + sta.mac + ") to wlan0")
+        env.agents[0].radios[0].vaps[0].associate(sta)
+
+        # send beacon query request (please take a look at https://github.com/prplfoundation/prplMesh/issues/1272)
+        debug("Sending beacon report query to repeater:")
+        request = '{mac} 0x73 0xFFFFFFFFFFFF 0x02 0x00 0x01 0x03 0x73 0x24 0x30'.format(mac=sta.mac)
```

Comment:
You did the "investigation is needed" part of #1272 
When I work I see "things" along the way, things that I know a second look / investigation is needed for but are not in the main focus at the time they were spotted. It happens all the time and it is inevitable. I try not to drop them. I describe them the best I can, spending less time then it would take to deeply understand or solve the issue and continue.
This is one of those cases.

Reply:
Problem solved, see #1272 for details.